### DISCUSSION TITLE
插件优化建议

### DISCUSSION THREAD

**Discussion by MrGongqi** [MuiseDestiny/zotero-style#161] [Ideas]
我觉得亲可以优先优化一下算法，把这个插件优化一下，让它尽量少占点内存。我知道这不是一下子就能做好的，但希望您可以多花点时间进行优化，谢谢。下面附图
![image](https://user-images.githubusercontent.com/49467246/220270076-b69f70d7-d984-4637-a267-15c7c6e13129.png)
图中是关闭关系图谱后的内存占有为727MB，而之前版本的插件（1.4.7）安装后内存占有为：530MB.
这只是我的一个小小的建议，希望您抽空可以加点时间优化。谢谢

**Comment by MuiseDestiny** [comment]
好的，应该是注册的列太多了，有时间我试试把不需要的功能彻底关闭试试。

**Comment by MrGongqi** [comment]
谢谢

**Reply by MuiseDestiny** [reply]
除了占用高，使用过程中卡顿明显吗？

**Reply by zhuyuhua23** [reply]
明显啊，卡顿啊，求更新解决啊

**Reply by zhuyuhua23** [reply]
> 明显啊，卡顿啊，求更新解决啊

**Comment by MuiseDestiny** [comment]
转化成讨论了，任何有关可以提升性能建议可以在这里提。

**Reply by zhuyuhua23** [reply]
明显卡顿啊，求更新解决啊

**Reply by MuiseDestiny** [reply]
关闭嵌套标签

**Comment by MuiseDestiny** [comment]
![image](https://user-images.githubusercontent.com/51939531/220273823-7fef87f7-0f0b-4fc1-ae13-4fbefa23a0df.png)
我的占用
